prog/weaveutil/datapath.go

```diff
@@ -3,16 +3,36 @@ package main
 
 import (
 	"fmt"
+	"net"
 	"os"
+	"strconv"
+	"syscall"
 
 	"github.com/weaveworks/weave/common/odp"
+	wnet "github.com/weaveworks/weave/net"
 )
 
 func createDatapath(args []string) error {
-	if len(args) != 1 {
-		cmdUsage("create-datapath", "<datapath>")
+	var (
+		mtu    int
+		err    error
+		setMTU bool
+	)
+
+	if len(args) != 1 && len(args) != 3 {
+		cmdUsage("create-datapath", "[--mtu <mtu>] <datapath>")
```

Comment:
TRTTD there is to introduce a separate command (`check-datapath`?) for the 2nd case.